src/nv_ingest/util/vdb/milvus.py

```diff
@@ -0,0 +1,196 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: LicenseRef-NvidiaProprietary
+#
+# NVIDIA CORPORATION, its affiliates and licensors retain all intellectual
+# property and proprietary rights in and to this material, related
+# documentation and any modifications thereto. Any use, reproduction,
+# disclosure or distribution of this material and related documentation
+# without an express license agreement from NVIDIA CORPORATION or
+# its affiliates is strictly prohibited.
+
+# pylint: skip-file
+
+import json
+import os
+import logging
+
+from fastapi import HTTPException
+from pymilvus import Collection, MilvusClient, DataType, connections, utility, BulkInsertState
+from pymilvus.bulk_writer import RemoteBulkWriter, BulkFileType
+from typing import List
+
+logger = logging.getLogger(__name__)
+
+
+# We are using UUIDs for the collection_name. However, Milvus collection names
```

Comment:
Please add unit tests as possible

Reply:
Removed this files as it is no longer required to satisfy this PR